api/db/repositories/productRepository.js

```diff
@@ -59,31 +61,34 @@ exports.deleteProduct = (productId) => {
     });
 };
 
-exports.getAll = () => {
-    return new Promise((resolve, reject) => {
-        Products.findAll({ include: [Products.Price] })
-            .map(el => el.get({ plain: true }))
-            .then((products) => {
-                resolve(products);
-            })
-            .catch((error) => {
-                reject(error)
-            });
-    });
+exports.getAll = async () => {
+    let allProducts;
+
+    try {
+        allProducts = await Products.findAll({ include: [Products.Price] }).map(el => el.get({ plain: true }));
```

Comment:
can you break this line so it's not that long and easier to read?

Reply:
Can try